topics/chip-seq/metadata.yaml

```diff
@@ -22,110 +22,110 @@ maintainers:
   - friedue
 
 references:
```

Comment:
Should we try to remove them here too and move them to the tutorial level?

Reply:
Yes, definitely, just a lot where it wasn't (yet) obvious which tutorial it belonged to or how/where to insert in the text.